src/components/Login.js

```diff
@@ -99,31 +107,52 @@ export default function Login() {
     }
 
     return (
-      <Form onSubmit={submitOtp}>
-        <Form.Group size="lg">
-          <Form.Control
-            autoFocus
-            type="text"
-            placeholder="Enter Auth Code"
-            value={otp}
-            onChange={(e) => setOtp(e.target.value)}
-          />
-        </Form.Group>
-        <Button block size="lg" type="submit" disabled={!validAuthCode(otp)}>
-          Login
-        </Button>
-      </Form>
+      <form onSubmit={submitOtp}>
+        <input
+          id="otp"
+          autoFocus
+          required
+          type="text"
+          placeholder="Enter Auth Code"
+          value={otp}
+          onChange={(e) => setOtp(e.target.value)}
+          className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 focus:outline-none focus:shadow-outline"
+        />
+        <p className="text-red-500 mt-2 text-xs italic">
+          {otp && !validAuthCode(otp) ? "Invalid auth code" : ""}
+        </p>
+        <button
+          type="submit"
+          disabled={!validAuthCode(otp)}
+          className="bg-blue-400 hover:bg-blue-500 text-white font-bold p-2 my-4 w-full border border-blue-500 rounded disabled:opacity-50"
+        >
+          {userLoginLoading ? "Loading..." : "Login"}
+        </button>
+      </form>
     )
   }
 
   return (
-    <div>
-      <Header />
-      <div className="Login">
-        {!otpGenerated && <PhoneForm />}
-        {otpGenerated && <OTPForm />}
-        {(requestAuthCodeLoading || userLoginLoading) && <p>Loading...</p>}
+    <div className="flex items-center min-h-screen p-6 bg-gray-50">
+      <div className="flex-1 h-full max-w-2xl mx-auto overflow-hidden bg-white rounded-lg shadow-xl">
+        <div className="flex flex-col overflow-y-auto md:flex-row">
+          <div className="flex h-32 md:h-auto md:w-1/2">
+            <img
+              src={process.env.PUBLIC_URL + "/logo.png"}
+              alt="Bitcoin Beach logo"
+              className="m-auto filter invert"
+            />
+          </div>
+          <main className="p-6 sm:p-12 md:w-1/2">
+            <h1 className="block text-3xl font-extrabold text-gray-900">Welcome</h1>
+            <div className="py-4">
+              {!otpGenerated && <PhoneForm />}
```

Comment:
Similar to the above comment about ! in front of conditionals, could we change this to make it xxx===yyy?